benchmark/operations/create_container.py

```diff
@@ -0,0 +1,319 @@
+# Copyright 2016 ClusterHQ Inc.  See LICENSE file for details.
+"""
+Operation to create a container.
+"""
+
+from functools import partial
+from itertools import repeat
+import random
+from uuid import UUID, uuid4
+
+from pyrsistent import PClass, field
+from zope.interface import implementer
+
+from flocker.apiclient import MountedDataset
+from flocker.common import gather_deferreds, loop_until
+from flocker.control import DockerImage
+
+from .._interfaces import IProbe, IOperation
+
+
+class EmptyClusterError(Exception):
+    """
+    Exception indicating that the cluster contains no nodes.
+    """
+
+
+def create_dataset(
```

Comment:
I'm going to leave it as is for now, since these top-level functions will probably get moved to a separate module to help implement the scenario in https://clusterhq.atlassian.net/browse/FLOC-3807